apps/explorer/lib/explorer/exchange_rates/source/coin_gecko.ex

```diff
@@ -216,7 +216,7 @@ defmodule Explorer.ExchangeRates.Source.CoinGecko do
     Application.get_env(:explorer, __MODULE__, [])[key]
   end
 
-  defp token_symbol_to_id_mapping_to_get_price(symbol) do
+  defp bridged_token_symbol_to_id_mapping_to_get_price(symbol) do
     case symbol do
       "UNI" -> "uniswap"
```

Comment:
interesting looks like a way to specify a particular coingecko id to use to fetch for a particular token @leonprou